tests/D2L.CodeStyle.TestAnalyzers.Tests/IgnoreAttribute/IgnoreAnalyzerTests.cs

```diff
@@ -37,6 +37,24 @@ class Test {
 		}
 
 		[Test]
+		public void DocumentWithoutTestIgnore_NoDiag() {
+			const string test = @"
+	using System;
+
+	namespace test {
+		[ReflectionSerializer]

```

Comment:
Did you forget to push? :/
I think a test to make sure ignore doens't matter for non-test methods is cool, because it won't compile with nunit3 (reason is required), but the `ReflectionSerializer` part shouldn't matter.